core/lib/multistore.ts

```diff
@@ -49,28 +52,54 @@ export function currentStoreView (): StoreView {
   return rootStore.state.storeView
 }
 
+export function getFinalStoreviewConfigInMultistoreMode (storeView: StoreView): StoreView {
```

Comment:
Can we come up with something smaller and more descriptive if function is exported member?

Reply:
Well... This function doesn't need to be exported really. I can make it private. Regarding the naming, do you have any recommendations?

Reply:
maybe `extendStoreView` ?

Reply:
@filrak I've changed the name to more descriptive getExtendedStoreviewConfigInMultistoreMode but I don't see a way to make it shorter without making the code less self-explaining.

Reply:
imho `inMultstoreMode` is redundant since it's explicit. There are no StoreViews without multistore

Reply:
Also this function is not a getter, so as @pkarw proposed it should be `extendStoreview`